editor/src/components/canvas/canvas-strategies/strategies/reparent-as-static-strategy.tsx

```diff
@@ -98,3 +116,140 @@ function getIdAndNameOfReparentToStaticStrategy(targetLayout: 'flex' | 'flow'):
       assertNever(targetLayout)
   }
 }
+
+export function applyStaticReparent(
```

Comment:
this function is just moved without changes from reparent-strategy-helpers.
this was the only place it was called from, so it was pointless for this to live in a separate helper

Reply:
Should the `export` modifier be deleted?

Reply:
ah yes!